frontend/src/utils/methods.ts

```diff
@@ -23,3 +23,39 @@ export const toggleFromArray: <T>(
  */
 export const replaceQuotationMarks = (text: string) =>
   text.replaceAll('"', "'");
+
+/**
+ *
+ * @param name
+ * @param obj
+ * @returns
+ */
+export const traverseObject: (name: string, obj: object) => string = (
```

Comment:
Looks unused? Or will it be utilized later?

Reply:
Seems I added it to the wrong branch - it will be used when merged with the others:)